### PR TITLE
vtimer: Add forward declarations of debug functions

### PR DESCRIPTION
**Pull request by gebart**
It caused build errors because of implicit declaration and conflicting types.

    riot/sys/vtimer/vtimer.c: In function ‘vtimer_callback’:
    riot/sys/vtimer/vtimer.c:207:9: error: implicit declaration of function ‘vtimer_print’ [-Werror=implicit-function-declaration]
             vtimer_print(timer);
             ^
    riot/sys/vtimer/vtimer.c: In function ‘vtimer_print_short_queue’:
    riot/sys/vtimer/vtimer.c:424:5: error: implicit declaration of function ‘priority_queue_print’ [-Werror=implicit-function-declaration]
         priority_queue_print(&shortterm_priority_queue_root);
         ^
    riot/sys/vtimer/vtimer.c: At top level:
    riot/sys/vtimer/vtimer.c:431:6: warning: conflicting types for ‘vtimer_print’
     void vtimer_print(vtimer_t *t)
          ^
    riot/sys/vtimer/vtimer.c:207:9: note: previous implicit declaration of ‘vtimer_print’ was here
             vtimer_print(timer);
             ^
    cc1: some warnings being treated as errors
    riot/Makefile.base:60: recipe for target 'riot/tests/vtimer_msg/bin/mulle/vtimer/vtimer.o' failed
    make[3]: *** [riot/tests/vtimer_msg/bin/mulle/vtimer/vtimer.o] Error 1

### CODE REVIEW

**Comment by gebart** [comment]
(the bug is only visible with ENABLE_DEBUG 1 in vtimer.c)

File: sys/vtimer/vtimer.c [hunk 1]
```diff
@@ -32,9 +34,16 @@
 
 #include "vtimer.h"
 
-#define ENABLE_DEBUG    (0)
```
Comment (PeterKietzmann) [inline sys/vtimer/vtimer.c]: @gebart why did you move this macro upwards? In most scripts you have this line and the `include "debug.h"` one below the other.

Comment (PeterKietzmann) [inline sys/vtimer/vtimer.c]: Ah now I see why you did that. Would you mind moving the include also upwards? (Sorry, I know this is a bit picky :-) )

**Comment by PeterKietzmann** [comment]
The above comment is trivial. In general: ACK

**Comment by gebart** [comment]
@PeterKietzmann comment addressed and squashed, waiting for Travis

**Comment by PeterKietzmann** [comment]
ACK and go
